### PR TITLE
Add support for F34

### PR DESCRIPTION
**Pull request by VladimirSlavik**
SSIA

### CODE REVIEW

File: anaconda_updates/anaconda_updates/releases/f34.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from . import GeneralBranch, Branch
+
+
+class Fedora34Branch(GeneralBranch):
+
+    def __init__(self):
+        super().__init__(branch_type=Branch.fedora34,
+                         cmd_args=["-f34", "--fedora34"],
+                         help="working on Fedora 34",
+                         version_script_params=["-f34", "-p"])
```
Comment (poncovka) [inline anaconda_updates/anaconda_updates/releases/f34.py]: Please, add `site_packages="./usr/lib/python3.9/site-packages/"`.

**Review by poncovka — APPROVED** [review]
Otherwise, it looks good to me.

File: anaconda_updates/anaconda_updates/releases/f34.py (hunk 1 above)
Comment (VladimirSlavik) [reply anaconda_updates/anaconda_updates/releases/f34.py]: Thank you! Added.

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Pull request merged**